addons/auth_signup/models/res_users.py

```diff
@@ -224,6 +247,54 @@ def send_unregistered_user_reminder(self, after_days=5):
             template = self.env.ref('auth_signup.mail_template_data_unregistered_users').with_context(dbname=self._cr.dbname, invited_users=invited_users[user])
             template.send_mail(user, email_layout_xmlid='mail.mail_notification_light', force_send=False)
 
+    def _alert_new_device(self):
+        """ send alert by mail to user with values of the request """
+        for user in self.env.user:
+            if user.email:
+                template = self.env.ref('auth_signup.alert_new_device_email')
+                assert template._name == 'mail.template'
+                email_values = {
+                    'email_cc': False,
+                    'auto_delete': False,
+                    'message_type': 'user_notification',
+                    'recipient_ids': [],
+                    'partner_ids': [],
+                    'scheduled_date': False,
+                }
+                context = self._retreive_request_values()
+                email_values['email_to'] = user.email
+                template.sudo().with_context(context).send_mail(user.id, raise_exception=True, email_values=email_values)
+                _logger.info("New device alert email sent for user <%s> to <%s>", user.login, user.email)
+
+    def _retreive_request_values(self):
+        #Collects time format "Day, Month dd, yyyy - hh:mm:ss (GMT)",
+        #location format "City, Region, Country", IP, Browser and OS
+        #from the request and return a list with their values.
+
+        date = datetime.now().strftime("%A, %B %d, %Y, %H:%M:%S (GMT)")
+        values = {
+            'date':date,
+            'location':'unknown city, unknown region, unknown country',
```

Comment:
```suggestion
            'location':False,
```
(cf comment in the template)